.github/workflows/task_e2e_tests.yml

```diff
@@ -93,4 +93,4 @@ jobs:
         if: always()
         with:
           name: videos-${{ runner.os }}
-          path: ./frontend/app/tests/e2e/videos
\ No newline at end of file
+          path: ./frontend/app/tests/e2e/videos
```

Comment:
Is the change here whitespace related?

Reply:
let me remove it, so, I made a change on the file and reverted, but prettier had already added the space